TomatoTomato2/Views/AddView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  AddView.swift
+//  TomatoTomato2
+//
+//  Created by Maria Kharybina on 06/01/2023.
+//
+
+import SwiftUI
+
+struct AddView: View {
+    
+    @State var textFieldText: String = ""
+    @State var sizePickerSelection: String = "L"
+    @State var typePickerSelection: String = "Research"
+    
+    var sizePickerOptions: [String] = ["XS", "S", "M", "L", "XL"]
```

Comment:
please move outside of VIew. 
even sth like this: `let sizes = ["XS", "S", "M", "L", "XL"]` on line 11, and `var sizePickerOptions: [String] = sizes`